ts/components/screens/BaseHeader.tsx

```diff
@@ -211,9 +224,11 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
               <IconFont name={customRightIcon.iconName} />
             </ButtonDefaultOpacity>
           )}
-        {!this.props.avoidNavigationEventsUsage && (
-          <NavigationEvents onDidFocus={this.handleFocus} />
-        )}
+        {fromNullable(this.props.accessibilityEvents).fold(
+          true,
+          ({ avoidNavigationEventsUsage, disableAccessibilityFocus }) =>
+            !(avoidNavigationEventsUsage && disableAccessibilityFocus)
```

Comment:
this condition causes a crash:

- avoidNavigationEventsUsage = true , disableAccessibilityFocus = false -> true (error)


fix and refactor 1068bdfe0ed427b0427ff8fd3256450350d47a48